gamemodes/terrortown/entities/weapons/weapon_tttbase.lua

```diff
@@ -9,6 +9,8 @@ local util = util
 local IsValid = IsValid
 local surface = surface
 local draw = draw
+local CLIENT = CLIENT
+local SERVER = SERVER
```

Comment:
localizing globals vars (no functions / tables) isn't the sense of this micro-optimization, especially regarding the SERVER and CLIENT var. If you really wanna do micro-optimization, then a seperated CLIENT and SERVER function would make more sense. This doesn't take any effect otherwise imho.